backend/src/utils/index.ts

```diff
@@ -1,19 +1,37 @@
-import express from 'express';
+import express, { NextFunction } from 'express';
 import { ErrorHandler } from '../services/error';
-import { Fields } from 'formidable';
-
-export const extractPayload = async (payload: express.Request): Promise<never | Fields | undefined> => {
+import { User } from './interfaces';
+export const extractPayload = async (payload: express.Request): Promise<never | Record<string, any> | undefined> => {
   try {
-    const isPayload = payload.fields !== undefined;
+    const isPayload = payload.body !== undefined;
     if (isPayload) {
-      return payload.fields as Fields;
+      return payload.body as Record<string, any>;
     } else {
       throw new ErrorHandler(400, { status: 'No Payload' });
     }
   } catch (e) {
     throw new ErrorHandler(400, { status: 'No Payload' });
   }
 };
+
+export const userExists = async (user: User): Promise<boolean> => {
+  return true;
+};
+
+export const checkAuthenticated = (req: Express.Request, res: Express.Response, next: NextFunction) => {
+  if (req.isAuthenticated()) {
+    return next();
+  }
+  return true;
```

Comment:
should this method return true if the request is not authenticated?

Reply:
The value should not matter, but it makes more sense to have it as a false, so i'll change it.